src/drivers/drv_hrt.h

```diff
@@ -109,6 +109,15 @@ __EXPORT extern hrt_abstime hrt_elapsed_time(const volatile hrt_abstime *then);
  */
 __EXPORT extern hrt_abstime hrt_store_absolute_time(volatile hrt_abstime *now);
 
+#ifdef __PX4_QURT
+/**
+ * Set a time offset to hrt_absolute_time on the DSP.
+ * @param time_diff_us: time difference of the DSP clock to Linux clock.
+ *   This param is positive because the Linux clock is ahead of the DSP one.
```

Comment:
@johnfinley fixed in https://github.com/PX4/Firmware/pull/4217/commits/fbd7c73043b184fbae0e94f3a2053cafdd89a145